gpflow/base.py

```diff
@@ -307,6 +304,18 @@ def _run_op(a, *args, **kwargs):
 Parameter._OverloadAllOperators()
 tf.register_tensor_conversion_function(Parameter, lambda x, *args, **kwds: x.read_value())
 
+TensorLike = Union[np.ndarray, tf.Tensor, tf.Variable, Parameter]
+"""
+Type alias for tensor-like types that are supported by most TensorFlow, NumPy and GPflow operations.
+
+NOTE: Does not work with multipledispatch dispatchers. See `TensorLikeTypes`.
+"""
+
+# We've left this as object until we've tested the performance consequences of using the full set
+# see github issue #1434
+TensorLikeTypes = (object,)  # (np.ndarray, tf.Tensor, tf.Variable, Parameter)
+""" Collection of tensor-like types for use with multipledispatch dispatchers.  """
```

Comment:
```suggestion
#: Collection of tensor-like types for registering implementations with multipledispatch dispatchers.
TensorLikeTypes = (object,)  # (np.ndarray, tf.Tensor, tf.Variable, Parameter)
# We've left this as object until we've tested the performance consequences of using the full set
# see https://github.com/GPflow/GPflow/issues/1434
```

Reply:
docstrings are usually below the item, no?

Reply:
strings after, comments before (or same line): https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-autodata

Reply:
ok. either way, is there a problem with using """ xyz """? I prefer that

Reply:
pycharm renders `"""` differently to `#:`, and I am (and I would wager others are) more used to `#` meaning comment not docstring, even if it has a `:` after it.

also multiline docstrings are common and easier with `"""`

Reply:
I prefer the comments but I don't mind very much :shrug:
I've left the content change as a new separate suggestion